tool/menu.lua

```diff
@@ -181,13 +182,32 @@ local function randomizermenu()
     resetfunction = randomizer.randomize
     state = STATES.NONE
 
-  elseif selected == 5 then -- Reset
+  elseif selected == 5 then -- Set Sequence
+    local sequence = nil
+    local check = false
+    local error = ""
+    while check == false do
+      local input = ui.input(error .. "Input Sequence")
+      sequence = nil
+      for i in string.gmatch(input, "%d+") do
+        table.insert(sequence, math.floor(i - 2))
```

Comment:
The `math.floor` shouldn't be needed here, should it?

Reply:
`math.floor` is there so that when the sequence is displayed on screen, it doesn't display as floats or doubles